README.md

```diff
@@ -68,7 +67,15 @@ $ curl http://api.example.com/v1/reviews/269
 - Use [Consumer Driven Contracts](https://martinfowler.com/articles/consumerDrivenContracts.html) to increase confidence when making API changes. Libraries used within the company for this are [Pact](https://docs.pact.io/) and [Mockingjay](https://github.com/quii/mockingjay-server)
 - Ensure all input values are sanitized, this will allow easier comparisons. e.g. `"Value1 !== value1"`
 
+## Media types
 
+APIs should default to return responses encoded as [JSON](https://www.json.org/json-en.html). Where APIs support multiple return formats they should use the `Accept` header for negotiation as defined by the HTTP spec, but this should not be required to return the default format.
+
+### Hypertext Application Language
+
+The JSON should follow the [JSON HAL (Hypertext Application Language)](https://tools.ietf.org/html/draft-kelly-json-hal-08) specification.
+
+In order to make the API consumable by any client that understands JSON and not require a HAL capable client, the `Content-Type` of the response should be `application/hal+json` only if the client sends that media type in the `Accept` request header. Otherwise the same content should be returned with `Content-Type: application/json`. Furthermore advanced HAL features like templated links should be avoided.
```

Comment:
Might want to explain why templated links should be avoided.

Reply:
Have added. Do you agree, or do you think we should embrace all of HAL?